Actions/CheckForUpdates/CheckForUpdates.HelperFunctions.ps1

```diff
@@ -560,3 +561,239 @@ function UpdateSettingsFile {
     }
     return $modified
 }
+
+function ResolveFilePaths {

```

Comment:
Some docs here wouldn't hurt :) Particularly around what "originalSourceFolder" is and what is expected to be in the files array